addon/components/dropdown-action.hbs

```diff
@@ -0,0 +1,9 @@
+<button
+    class="dropdown-action dropdown-item"
+    type="button"
+    {{on "click" this.onClick}}
+    disabled={{this.disabled}}
```

Comment:
I dont think we need `disabled` defined here, as it'll be passed through automatically via the `...attributes` since its a native HTML attribute.

You can then also remove the `disabled` from your component Args interface and the getter in the component.